cgi-bin/LJ/Web.pm

```diff
@@ -786,11 +786,15 @@ sub icons_for_remote {
                 push @pics, [ $res{"pickw_$i"}, $res{"pickwurl_$i"} ];
             }
             @pics = sort { lc( $a->[0] ) cmp lc( $b->[0] ) } @pics;
+            my $defaultpicurl =
+                  $res{defaultpicurl}
+                ? $res{defaultpicurl}
+                : $LJ::IMGPREFIX . $LJ::Img::img{nouserpic_sitescheme}->{src};
```

Comment:
nit: Simpler and more canonical to write,

```perl
my $defaultpicurl = $res{defaultpicurl} || $LJ::IMGPREFIX . $LJ::Img::img{nouserpic_sitescheme}->{src};
```

Reply:
ah, good call, thx.

Reply:
although make sure the precedence doesn't bite you ... I forget the precedence of `.` heh

Reply:
haha, I try to always use parens if there are boolean operators in the mix. Even if you get the precedence right, it's a mental DOS attack on whoever ends up reading the code next.